lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -138,7 +138,12 @@ sub dev_pipeline_json :Chained('dev_pipeline_base') :PathPart('json') :Args(0) {
 
     my @ias;
     my $key;
-    @ias = $rs->search({'dev_milestone' => { '=' => ['planning', 'development', 'testing', 'complete']}});
+    @ias = $rs->search(
```

Comment:
the future intentions for this code should be noted in a comment, why it's here, etc.